lib/presentation/extensions/client_extension.dart

```diff
@@ -27,4 +27,8 @@ extension ClientExtension on Client {
   String mxid(BuildContext context) => userID ?? L10n.of(context)!.user;
 
   String? get pusherNotificationClientIdentifier => userID?.sha256Hash;
+
+  bool supportMultipleAccountsInTheSameHomeserver(Uri thatHomeserver) {
+    return isLogged() && homeserver == thatHomeserver;
+  }
```

Comment:
```
  bool supportMultipleAccountsInTheSameHomeserver(Uri thatHomeserver) {
    return false;
  }
```
Move this function to AppConfig. Then combine check new homeserver with existed homeserver